src/core/collection/__tests__/Collection.test.tsx

```diff
@@ -53,6 +53,7 @@ describe('event selection module', () => {
 
     // No other cards than the ongoing events should be shown.
     // There is a linkbox that wraps the whole card and another link that wraps the arrow-icon
-    expect(screen.getAllByRole('link').length).toBe(activeEvents.length * 2);
+    // expect(screen.getAllByRole('link').length).toBe(activeEvents.length * 2);
+    expect(screen.getAllByRole('link').length).toBe(activeEvents.length);
```

Comment:
@melniiv is this ok? Are the links really reduced in the latest version so that only 1 can be found by a screen reader.